doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -225,6 +228,20 @@ Cellular samples
 
   * Updated the :file:`.pem` certificate for example.com.
 
+* :ref:`nrf_cloud_rest_cell_pos_sample` sample:
+
+  * Removed:
+
+    * The button press interface for enabling the device location card on the nRF Cloud website. The card is now automatically displayed.
```

Comment:
You have confirmed this auto display works correctly I assume?

Reply:
yes